cmd/convox/templates/init/rails/docker-compose.yml

```diff
@@ -6,3 +6,5 @@ web:
   ports:
     - 80:4000
     - 443:4001
+  volumes:
```

Comment:
Ah good call on the COPY.

There's no reason why either strategy can't work. I think it's a matter of making the decision in a way that's actionable for the rails example to improve and for other language frameworks to get moving.